python/ray/autoscaler/command_runner.py

```diff
@@ -726,9 +735,24 @@ def run_init(self, *, as_head, file_mounts):
                 logger.error(f"A container with name {self.container_name} " +
                              f"is running image {running_image} instead " +
                              f"of {image} (which was provided in the YAML")
-
-        # Copy bootstrap config & key over
-        if as_head:
-            for copy_cmd in docker_autoscaler_setup(self.container_name):
-                self.run(copy_cmd, run_env="host")
+            mounts = self.run(
+                check_bind_mounts_cmd(self.container_name),
+                with_output=True,
+                run_env="host").decode("utf-8").strip()
+            try:
+                active_mounts = json.loads(mounts)
+                active_remote_mounts = [
+                    mnt["Destination"] for mnt in active_mounts
+                ]
+                for remote, local in file_mounts.items():
+                    remote = self._docker_expand_user(remote)
+                    if remote not in active_remote_mounts:
+                        cli_logger.error(
+                            "Please ray stop & restart cluster to "
+                            f"allow mount {remote}:{local} to take hold")
+                        cli_logger.old_error(
+                            "Please ray stop & restart cluster to "
+                            f"allow mount {remote}:{local} to take hold")
+            except json.JSONDecodeError:
+                pass
```

Comment:
logger.debug

Reply:
(or like cli_logger.verbose()) is more accurate.

Reply:
I'll switch to `verbose`, but I'm worried that users may want to know this error immediately if they 1) Setup a cluster and 2) try to add an additional `file_mount`.

Reply:
Sorry I miscommunicated - we should do a cli logger output if we catch this json decode error.

Reply:
Oh! Thanks for the clarification